donut/modules/bodfeedback/permissions.py

```diff
@@ -0,0 +1,9 @@
+import enum
```

Comment:
@davistardif this is the first feature since permissions was birthed to use permissions. can you check that it is fine under your design :)